docs/telemetry/PrivacyPolicy.md

```diff
@@ -0,0 +1,5 @@
+# Privacy Policy
+
+To be filled in by legal team based off of meeting documented at https://pl-strflt.notion.site/2023-01-05-Privacy-Policy-Updates-9bef84008b52449496ad5b806d529dfe
+
+Until then, for all project's other than ipfs-companion (which is the only project we own with a current [privacy policy](https://github.com/ipfs/ipfs-companion/blob/main/PRIVACY-POLICY.md)), you can reference [ipns://protocol.ai/legal/#privacy-policy](ipns://protocol.ai/legal/#privacy-policy)
```

Comment:
lol, https://protocol.ai/legal/#privacy-policy has no content – it is a copy-paste of terms of service above about CC license, not a single word about privacy :upside_down_face: 


ps. for the future, no reason to use native URI like this. Many people won't have `ipns://` handler and won't be able to read this.
Just use normal URL/gateway one, and users with Brave/Copmpanion will be able to load it over IPFS anyway.

```suggestion
Until then, for all project's other than ipfs-companion (which is the only project we own with a current [privacy policy](https://github.com/ipfs/ipfs-companion/blob/main/PRIVACY-POLICY.md)), you can reference [protocol.ai/legal/#privacy-policy](https://protocol.ai/legal/#privacy-policy)
```

Reply:
resolved locally with update to ipfs-companion policy pushing up shortly.